dev/com.ibm.ws.st.liberty.tools/feature.xml

```diff
@@ -3,7 +3,8 @@
       id="com.ibm.ws.st.liberty.tools"
       label="%featureName"
       version="18.0.3.qualifier"
-      provider-name="%providerName">
+      provider-name="%providerName"
+      plugin="com.ibm.ws.st.core">
```

Comment:
This looks like a top level feature.  Usually we don't mention any plugins in top level features but just include other features.